src/ucp/wireup/select.c

```diff
@@ -233,10 +233,16 @@ ucp_wireup_check_keepalive(const ucp_wireup_select_params_t *select_params,
                            const char *title, const char **flag_descs,
                            char *reason, size_t max)
 {
-    ucp_worker_h worker = select_params->ep->worker;
+    ucp_worker_h worker              = select_params->ep->worker;
+    uint64_t required_ep_check_flags = UCT_IFACE_FLAG_EP_CHECK;
     char title_keepalive[128];
     char title_ep_check[128];
 
+    if (ucp_ep_init_flags_has_cm(select_params->ep_init_flags) &&
+        (flags & UCT_IFACE_FLAG_CONNECT_TO_IFACE)) {
+        required_ep_check_flags |= UCT_IFACE_FLAG_AM_BCOPY;
```

Comment:
1. pls add comment it's needed to send keepalive over amctive messages
2. rename required_ep_check_flags  -> iface_flags